packages/uploadthing/src/sdk/index.ts

```diff
@@ -435,15 +436,14 @@ export class UTApi {
     const program = Effect.gen(function* () {
       const { apiKey, appId } = yield* UTToken;
       const ufsHost = yield* UfsHost;
-
+      const doAppID = Boolean(yield* DoAppIDInUrls);
       const proto = ufsHost.includes("local") ? "http" : "https";
-      const ufsUrl = yield* generateSignedURL(
-        `${proto}://${appId}.${ufsHost}/f/${key}`,
-        apiKey,
-        {
-          ttlInSeconds: expiresIn,
-        },
-      );
+      const urlBase = doAppID
+        ? `${proto}://${appId}.${ufsHost}/f/${key}`
+        : `${proto}://${ufsHost}/f/${key}`;
```

Comment:
can you use the `/a/APP_ID/KEY` format instead? And then rename the config var to something like (naming is hard...) `URL_APPID_LOCATION="subdomain" | "path"`